fastchat/serve/inference.py

```diff
@@ -4,13 +4,15 @@
 import json
 import math
 import os
+import requests
 import sys
 import time
 from typing import Iterable, Optional, Dict
 import warnings
 
 import psutil
 import torch
+from io import BytesIO
```

Comment:
Organize the imports as three blocks

```
build-in libraries

third-party libraries

fastchat libraries
```

In each block, sort the imports alphabetically.

In this case, `requests` should go to the second block and `io` should go to the first block